ci/jenkinsfile.groovy

```diff
@@ -134,6 +152,45 @@ stage('Build') {
           sh(script: "${pkg_cpu} conda run -n py38 ./ci/task/build_wheel.sh", label: 'Build MLC LLM wheel')
           sh(script: "${pkg_cpu} conda run -n py38 ./ci/task/build_clean.sh", label: 'Clean up after build')
           sh(script: "ls -alh ./wheels/", label: 'Build artifact')
+          pack_lib('mlc_wheel_vulkan', 'wheels/*.whl')
+        }
+      }
+    }
+  )
+}
+
+stage('Model Compilation') {
+  parallel(
+    'CUDA': {
+      node('CPU-SMALL') {
+        ws(per_exec_ws('mlc-llm-compile-cuda')) {
+          init_git(false)
+          sh(script: "ls -alh", label: 'Show work directory')
+          unpack_lib('mlc_wheel_cuda', 'wheels/*.whl')
+          sh(script: "${run_cuda} conda env export --name ci-unittest", label: 'Checkout version')
+          sh(script: "${run_cuda} conda run -n ci-unittest ./ci/task/test_model_compile.sh", label: 'Testing')
+        }
+      }
+    },
+    'ROCm': {
+      node('CPU-SMALL') {
+        ws(per_exec_ws('mlc-llm-compile-rocm')) {
+          init_git(false)
+          sh(script: "ls -alh", label: 'Show work directory')
+          unpack_lib('mlc_wheel_rocm', 'wheels/*.whl')
+          sh(script: "${run_rocm} conda env export --name ci-unittest", label: 'Checkout version')
+          // sh(script: "${run_rocm} conda run -n ci-unittest ./ci/task/test_model_compile.sh", label: 'Testing')
```

Comment:
per discussion with @MasterJH5574: we will enable ROCm/Vulkan tests once related bugs are fixed